backend/fleet_management/models.py

```diff
@@ -33,3 +33,8 @@ class Route(models.Model):
     end_mileage = models.IntegerField(null=False)
     fuel_level_begin = models.FloatField(null=False)
     fuel_level_end = models.FloatField(null=False)
+
+    def fuel_consumption(self):
```

Comment:
Please make it either a property by decorating with `@property` OR change method name to `get_fuel_consumption`. It will be easier to reason whether it's a prop or method by name this way.

Reply:
I didn`t know that. Thanks.